packages/subgraph/src/utils/tokens/erc1155.ts

```diff
@@ -198,88 +198,132 @@ export function handleERC1155Action(
   proposalId: string,
   actionIndex: number,
   event: ethereum.Event
-): void {
+): bool {
   let contract = fetchERC1155(token);
   if (!contract) {
-    return;
+    return false;
   }
 
   let functionSelector = data.toHexString().substring(0, 10);
```

Comment:
We can start using the `getMethodSignature`defined in the utils to get this

Reply:
addressed in [16b8444](https://github.com/aragon/osx/pull/472/commits/16b84446211c26f20054618fedbec9ad87504365)